sapmon/payload/provider/base.py

```diff
@@ -208,27 +208,27 @@ def __init__(self,
       self.fullName = "%s.%s" % (self.providerInstance.fullName, self.name)
       self.tracer = providerInstance.tracer
 
+   # Return check name for locking
+   def getLockName(self) -> str:
+      return "%s/%s" % (self.providerInstance.fullName, self.name)
```

Comment:
```suggestion
      return "%s.%s" % (self.providerInstance.fullName, self.name)
```
`self.providerInstance.fullName` is `"%s/%s" % (self.providerType, self.name)` (line 42).
therefore, I would suggest to replace "/" with "." to avoid duplicate "/"

Reply:
done